lib/chef/provider/package/portage.rb

```diff
@@ -25,6 +25,8 @@ class Chef
   class Provider
     class Package
       class Portage < Chef::Provider::Package
+
+        provides :package, platform: "gentoo"
         provides :portage_package
```

Comment:
It doesn't really matter.  We probably do need to decide if we care or not, but right now there's no rule to follow.